.circleci/ansible/bin/install_dependencies.sh

```diff
@@ -11,6 +11,6 @@ if [ ! -e /sbin/vboxconfig ]; then
   sudo sh -c "echo 'deb http://download.virtualbox.org/virtualbox/debian xenial contrib non-free' > /etc/apt/sources.list.d/virtualbox.list"
   wget -q http://download.virtualbox.org/virtualbox/debian/oracle_vbox.asc -O- | sudo apt-key add -
   sudo apt-get update && sudo apt-get install -y --allow-unauthenticated virtualbox-5.2
-  sudo apt-get install -y linux-headers-generic linux-headers-4.4.0-112-generic
+  sudo apt-get install -y linux-headers-generic linux-headers-$(uname -r)
```

Comment:
That makes sense now :+1: 

But shouldn't headers install happen before installing virtualbox?

Reply:
Likely. I wanted to try this change first. I'll change the order in the next commit.

Reply:
:100: That's a good way of doing it!

Reply:
The `linux-headers` are not required to install `virtualbox-5.2`.

Reply:
Why are we installing them then?

Reply:
It fails if I remove that line. See 0e4cff1. I'll now try with it prior to the installation of `virtualbox-5.2`, as discussed yesterday.